packages/sqlite_async/lib/src/sqlite_connection.dart

```diff
@@ -76,6 +76,10 @@ abstract interface class SqliteWriteContext extends SqliteReadContext {
   /// parameter set.
   Future<void> executeBatch(String sql, List<List<Object?>> parameterSets);
 
+  // Execute a query that potentially contains multiple statements.
+  Future<void> executeMultiple(String sql,
+      [List<Object?> parameters = const []]);
```

Comment:
For `executeMultiple`, we should probably remove the `parameters` argument like you mentioned in the issue.